api.go

```diff
@@ -272,6 +272,8 @@ func Flush() error {
 	}
 	if !is_cursor_hidden(cursor_x, cursor_y) {
 		write_cursor(cursor_x, cursor_y)
+	}else {
+		outbuf.WriteString(funcs[t_hide_cursor])
```

Comment:
Can this be `if funcs[t_hide_cursor] then call that function, else write the cursor to cursor_x, cursor_y`?

Reply:
i pulled the code right from the hide cursor func. I could call that instead?

Reply:
Nah, this looks good. I was thinking about if `funcs[t_hide_cursor]` could be empty if the terminal didn't support hiding the cursor without looking at the code. There are other places in the code where this is called without additional checks, so it seems like it's always going to be something valid :)

Reply:
ah I see :)